content/en/getting_started/application_security/_index.md

```diff
@@ -24,72 +24,84 @@ further_reading:
 
-2. **Enable ASM on the most exposed services.** Click `Enable ASM` in the ASM Status column to see the instructions or share them with the service's owner. 
+   **Note**: If no vulnerabilities or suspicious requests are reported:
 
-   Because ASM relies on the same library as APM, configuring a single environment variable is all that is needed to get ASM enabled on a service that is already sending traces. Read more in the [Enabling ASM documentation][2].
+   - Ensure your services are using a recent Datadog tracing library version. From the [APM Service Catalog][2]: Open any service, navigate to: **Sidepanel > Tracing Configuration**. 
 
-3. **Explore your first suspicious requests.** Go to **[Security --> Application Security][7]** to see suspicious requests listed in ASM.
+     {{< img src="getting_started/appsec/ASM_Tracing_Configuration.png" alt="Tracer Configuration tab in APM Service Catalog page view. Highlighting which version of the Datadog Agent, and Datadog tracing library are being used by your services." style="width:100%;" >}}
 
-   - If the client IP is not well-resolved (for example, if it's showing an internal or proxy IP), [configure the client IP header][3].
+2. After you have identified a few services under security risks, proceed to [enable ASM][20].
 
-   - Open the **[Traces][8]** page and click one of the traces for a suspicious request. The Attack Flow diagram helps you understand all the services hit by the attack. Scroll down to the Security section to read about why this request was flagged as suspicious. Read more in [How Application Security Management Works][4].
+## Setup ASM
 
-   You don't need to examine every suspicious request individually. ASM generates _signals_ when your attention is required, which you can learn about in Phase 2.
+ASM relies on the same library as Datadog Application Performance Monitoring (APM). 
+To benefit from all the ASM capabilities such as detections, automatic updates, and protection, first enable [Remote Configuration][15].
 
-4. Now that ASM is set up detecting suspicious requests on your service in real-time, **let the product run for a couple days**.
+### Enable ASM with in-app instructions
 
-If your services use recent versions of tracing libraries, ASM immediately [detects security vulnerabilities][13] in the upstream libraries that your services have as dependencies, and APM previews this information for you in its Security view. Optionally, explore ASM's consolidated [Vulnerabilities view][14] to see how it helps you triage and prioritize vulnerabilities.
+Go to the [ASM Setup page][1] and follow the instructions to get started. This includes:
+- Guided selection of services that would benefit from ASM.
+- Configuring your Datadog tracing libraries with an environment variable.
+- Restarting your services.
```

Comment:
Move this section to after Enable RC? Or just delete?

Reply:
1. In-APP instructions
2. Activation through RC